src/MaxwellSolvers/Maxwell.h

```diff
@@ -18,46 +18,35 @@ namespace ippl {
     template <typename EMField, typename SourceField>
     class Maxwell {
```

Comment:
No inheritance from SolverBase?
```mermaid
graph TB

X[SolverBase] --> B(Electrostatics)
X[SolverBase] --> A(Maxwell)
B[Electrostatics] --> C[Poisson]
B[Electrostatics] --> FFT
B[Electrostatics] --> FEM
A(Maxwell) --> D(FDTD)
A(Maxwell) --> E(FEM)
```

Reply:
I feel SolverBase would be over design. However, we can discuss this later.